docs/pages/index.tsx

```diff
@@ -1,33 +1,73 @@
-import { MDXRemote } from 'next-mdx-remote';
-import { normalize } from 'path';
-
-import { getMarkdownData, serializeMarkdown } from '../lib/mdxUtils';
-import { mdxComponents } from '../components/utils';
+import { Columns, Column } from '@ag.ds-next/columns';
+import { Content } from '@ag.ds-next/content';
+import { Heading } from '@ag.ds-next/heading';
+import { Stack } from '@ag.ds-next/box';
+import { Text } from '@ag.ds-next/text';
+import { TextLink } from '@ag.ds-next/text-link';
+import { tokens } from '@ag.ds-next/core';
 import { AppLayout } from '../components/AppLayout';
-import { Body } from '@ag.ds-next/body';
-import { PageLayout } from '../components/PageLayout';
-
-type StaticProps = Awaited<ReturnType<typeof getStaticProps>>['props'];
+import { PictogramCard } from '../components/PictogramCard';
 
-export default function Homepage({ source }: StaticProps) {
+export default function Homepage() {
 	return (
 		<AppLayout>
-			<PageLayout editPath="/docs/README.md">
-				<Body>
-					<MDXRemote {...source} components={mdxComponents} />
-				</Body>
-			</PageLayout>
+			<Content background="bodyAlt">
+				<Stack maxWidth={tokens.maxWidth.bodyText} gap={1.5}>
```

Comment:
technically, you shouldn't need the maxWidth here as it's baked in to the Text component